src/dependencies/bootstrap/Gemfile

```diff
@@ -0,0 +1,12 @@
+# Ruby Gems for building and testing Bootstrap
+# Run `grunt update-gemfile-lock` to update to the latest compatible versions
+
+source 'https://rubygems.org'
```

Comment:
why do we need ruby? Do you need it to install dependencies? Please make ASCII list of dependencies and we should start writing spec files for them.